src/NotificationTargetCommonITILObject.php

```diff
@@ -1678,6 +1701,96 @@ public function getDataForObject(CommonDBTM $item, array $options, $simple = fal
         return $data;
     }
 
+    protected function getActorData(CommonDBTM $actor, int $actortype, string $key_prefix): array
+    {
+        $data = [
+            sprintf('##%s.itemtype', $key_prefix)  => get_class($actor),
```

Comment:
Ending `##` are missing.
```suggestion
            sprintf('##%s.itemtype##', $key_prefix)  => get_class($actor),
```